qiskit/circuit/quantumcircuit.py

```diff
@@ -1670,6 +1670,7 @@ def draw(
         initial_state: bool = False,
         cregbundle: bool = None,
         wire_order: list = None,
+        encoding: str = None,
```

Comment:
Strictly this is a new feature, so would make this PR ineligible for backport.  I don't actually think it's related to the meat of this PR, though - could we split it off?

Reply:
Done in https://github.com/Qiskit/qiskit/pull/10842/commits/e1d7982f3177d70010e4c1e264d6c9fecd6aedc5.